docs/environment-variables.md

```diff
@@ -18,4 +18,16 @@ When set to any value, enables detector experiments, a feature to compare the re
 same ecosystem. The available experiments are found in the [`Experiments\Config`](../src/Microsoft.ComponentDetection.Orchestrator/Experiments/Configs)
 folder.
 
+## `GRADLE_PROD_CONFIGURATIONS_REGEX`
+
+Enables dev-dependency categorization for the Gradle
+detector. Dependencies connected to Gradle configurations NOT matching
```

Comment:
This isn't quite sufficient level of granularity. A specific configuration wont be production across the board for all lockfiles in a project. Likely only one configuration in a handful of lockfiles is production and the rest are dev dependencies.

A java project will use the `classpath` configuration for example, but that would also apply to the buildscript runtime configuration.

And android project might have `releaseClasspath` as its production configuration, but a test app in the same repo would also have a `releaseClasspath` configuration.

Reply:
It would require a bit more work on the consumer's end, but if we were to add configuration for tagging specific lockfiles as production vs dev, we can setup production configurations to be written to a separate lockfile from dev dependencies.

Reply:
That's fair. I can expand the configuration to allow specification of matching for prod or dev lockfiles, but will wait to hear from the repo owners to resolve the env-var vs configuration-file question from the other comment thread.